pr-data.csv

```diff
@@ -4904,7 +5039,368 @@ https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.singleSslSocketFactory,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.singleVerifier,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.verifierAccepts,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
-https://github.com/kiegroup/optaplanner,dff7457cb5ba1557727deddef5b675dd00fbf8a8,optaplanner-benchmark,org.optaplanner.benchmark.config.PlannerBenchmarkConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,DeveloperFixed,,
```

Comment:
This test is not removed. The diff file shows deleted because I updated it to pass format checker (see line 5403).  Reason: after merging https://github.com/TestingResearchIllinois/idoft/pull/1460 , all `DeveloperFixed` entries have to contain a non-empty note. This test is `DeveloperFixed` but does not have a note, and the format checker fails without changing this line, because I added new entries that change its relative line number.  I did a binary search to find the latest commit that the test is no longer ID and put it in the note (line 5403), so the format checker passes.